libs/sdk-ui-filters/src/AttributeFilter/AttributeFilter.tsx

```diff
@@ -230,42 +228,55 @@ class AttributeFilterCore extends React.PureComponent<IAttributeFilterProps, IAt
-        } catch (error) {
-            this.setState({ title: "", error, isLoading: false });
-
-            this.props.onError(error);
-        }
-    };
-
-    private onApply = (selectedItems: IAttributeElement[], isInverted: boolean) => {
-        const useUriElements =
-            this.props.filter && isAttributeElementsByRef(filterAttributeElements(this.props.filter));
-
-        const filterFactory = isInverted ? newNegativeAttributeFilter : newPositiveAttributeFilter;
-
-        const filter = filterFactory(
-            getObjRef(this.props.filter, this.props.identifier),
-            useUriElements
-                ? { uris: selectedItems.map((item) => item.uri) }
-                : { values: selectedItems.map((item) => item.title) },
-        );
-
-        return this.props.onApply(filter);
-    };
-}
+    const { selectedItems, isInverted } = getInitialDropdownSelection();
+
+    return (
+        <IntlWrapper locale={locale}>
+            {attributeError ? (
+                <FilterError error={attributeError} />
+            ) : (
+                <MediaQuery query={MediaQueries.IS_MOBILE_DEVICE}>
+                    {(isMobile) => (
+                        <IntlTranslationsProvider>
+                            {(translationProps: ITranslationsComponentProps) => {
+                                return (
+                                    <AttributeDropdown
+                                        titleWithSelection={titleWithSelection}
+                                        displayForm={getObjRef(currentFilter, identifier)}
+                                        backend={backend}
+                                        workspace={workspace}
+                                        onApply={onFilterApply}
+                                        title={title || attributeTitle}
+                                        isInverted={isInverted}
+                                        selectedItems={selectedItems}
+                                        isLoading={
+                                            attributeStatus === "pending" || attributeStatus === "loading"
+                                        }
+                                        translationProps={translationProps}
+                                        isMobile={isMobile}
+                                        fullscreenOnMobile={fullscreenOnMobile}
+                                        parentFilters={getValidElementsFilters(
+                                            resolvedParentFilters,
+                                            parentFilterOverAttribute,
+                                        )}
+                                    />
+                                );
+                            }}
+                        </IntlTranslationsProvider>
+                    )}
+                </MediaQuery>
+            )}
+        </IntlWrapper>
+    );
+};
+
+AttributeFilterCore.defaultProps = {
+    locale: "en-US",
```

Comment:
This should not be hardcoded like this, there is a constant for this in @gooddata/sdk-ui called `DefaultLocale` please use that instead.